ggshield/verticals/secret/output/schemas.py

```diff
@@ -14,6 +14,8 @@ class FlattenedPolicyBreak(BaseSchema):
     incident_url = fields.String(required=True, dump_default="")
     incident_details = fields.Nested(SecretIncidentSchema)
     known_secret = fields.Bool(required=True, dump_default=False)
+    is_excluded = fields.Bool(required=False)
+    exclude_reason = fields.String(required=False)
```

Comment:
It's probably a bit late, but I just realized `exclude_reason` does not sound like correct English. I think it should be `exclusion_reason`.